dev/README.md

```diff
@@ -281,6 +281,13 @@ RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/s
         chmod +x ./kubectl && \
         mv ./kubectl /usr/local/bin/kubectl
 
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash && \
```

Comment:
You could maybe use an [ARG](https://docs.docker.com/reference/dockerfile/#arg) with a default for this version?

Calls out the dependency for the build better?

Reply:
Hmm, the real dependency is the latest LTS version of Node.

Reply:
I like the idea of making the versions visible at the beginning of the script.